insalan/user/models.py

```diff
@@ -147,9 +147,9 @@ def send_email_confirmation(user_object: User):
         send_mail(
             _("Confirmez votre courriel"),
             _("Confirmez votre adresse de courriel en cliquant sur ")
-            + "http://api."
+            + "http://"
             + getenv("WEBSITE_HOST", "localhost")
-            + reverse("confirm-email", kwargs={"user": user, "token": token}),
+            + reverse("verification", kwargs={"user": user, "token": token}),
```

Comment:
This reverse does not exist, I don't know what you were trying to do but `verification` is not listed in `users/url.py`, and so it will not be found by reverse. We can only solve this by hardcoding a pattern and filling in the blanks.